public/pages/AnomalyCharts/containers/AnomaliesChart.tsx

```diff
@@ -153,10 +149,14 @@ export const AnomaliesChart = React.memo((props: AnomaliesChartProps) => {
   const setUpAlertsButton = () => (
     <AlertsButton
       monitor={props.monitor}
-      detectorId={props.detectorId}
-      detectorName={props.detectorName}
-      detectorInterval={get(props, 'detectorInterval', 1)}
-      unit={get(props, 'unit', 'Minutes')}
+      detectorId={get(props.detector, 'id', '')}
+      detectorName={get(props.detector, 'name', '')}
+      detectorInterval={get(
+        props.detector,
+        'detectionInterval.period.interval',
+        1
+      )}
+      unit={get(props.detector, 'detectionInterval.period.unit', 'Minutes')}
     />
   );
```

Comment:
any particular reason why you didn't continue the removal of props to `AlertsButton` and `AnomalyHeatmapChart`?

Reply:
alert button is still needed. We need it to redirect user to Alerting page, but we won't show alerts info, as the alerts related UX is not finalized yet for HC detector.

Reply:
Sorry, I meant why you didn't also refactor the `AlertsButton` and `AnomalyHeatmapChart` detector props into a single `detector` prop as well.

Reply:
oh, I see. I think it is not very necessary to pass a whole detector info to it because AlertsButton needs only a few detector info, while `detector` has other info such as state, category field as well.

Reply:
That's fair, makes sense.